src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days >= 3 && days < 7) {
+    return (days * 40) - 20;
+  }
+
+  if (days >= 7) {
+    return (days * 40) - 50;
+  }
```

Comment:
```suggestion
  return (days * 40) - 50;
```
no need last if

Reply:
comment about constants for magic numbers not fixed
for example
 ```
  const dayCost = 40;
  const threeDaysDiscount = 20;
 ```
also make for 50, 3, 7. 
It make code more readable